src/ProjectNeon/Assets/Data/Cards/ElectroBlade/Jolt.asset

```diff
@@ -0,0 +1,112 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 6b41b1118d12e5c48a6e7ed020b96a84, type: 3}
+  m_Name: Jolt
+  m_EditorClassIdentifier: 
+  art: {fileID: 21300000, guid: d94974f5498aa47499865518f1d2180a, type: 3}
```

Comment:
This doesn't seem to be custom art for this card. 

When I have been doing these, I have been adding art. 

Either you can add a suitable art (I can get you access to some art assets to pick from), or you should open a puzzle for anything that still needs art, with time amount of 15 minutes, and a list of the art assets that need to be initialized.